### ISSUE TITLE
Can't load posts

### ISSUE THREAD

**Issue by doesnmisclown** [meower-media-co/Meower-Svelte#169]
## Expected behavior

Load posts after pressing skip
## Current behavior

Error with posts loaded
## Steps to reproduce

1. Open client and press skip
## Screenshots / Logs

![Screenshot_20230710-190601](https://github.com/meower-media-co/Meower-Svelte/assets/116358854/1417ec56-8d60-48e1-8502-1726dd2dc005)

## Platform info

**Meower Client**
- Meower Client Version + Build Date: Meower Svelte v1.6.2
- 

**Desktop/Mobile Device**
 - OS: [Examples: Windows, Linux, macOS...]
 - Version: [Examples: 22H2, 22.04, 13.7...]
 - RAM: [Examples: 8GB, 16GB, 32GB...]
 - Storage: [Examples: 128GB, 256GB...]
 - CPU Architecture: [Examples: x86, x86_64, PPC, ARM32, ARM64...]
 OS: Android
 Version: 10
 RAM: 4GB
Storage: 32GB
CPU: ARM64

## Did I read the instructions before making this bug report?

**I have read the instructions**

**Comment by tnix100** [comment]
Hey, looks like our API may have had a short outage, could you please try this again :)

**Comment by showierdata9978** [comment]
unable to repro 
![image](https://github.com/meower-media-co/Meower-Svelte/assets/68120127/3baf3ac7-18b4-4759-aa84-ab53ccd44f9a)

**Comment by JoshAtticus** [comment]
Hello, it appears that you are using Firefox, which has previously had known issues with Meower on mobile. Do you mind checking if Meower works on a Chromium-based web browser? (e.g. Chrome, Vivaldi, Opera).

If it does not work on a Chromium-based web browser, try disabling any ad-blocker extensions in your web browser and try loading Meower again.

If that does not fix the issue, check if you are using a VPN or Proxy that may be blocking Meower. If one is enabled, try turning it off and try again. This includes system-wide ad-blocking apps such as Blokada or AdGuard. To turn off VPN on AOSP, Stock Android, or Google Pixel, go to `Settings > Network and Internet > VPN > *Click on any VPNs labelled with "Connected"* > Disconnect`

In the case that Meower is still not loading on your device, head into the DNS settings on your phone, and disable Secure DNS/Private DNS. On AOSP, Stock Android, or Google Pixel, this is in `Settings > Network and Internet > Secure DNS/Private DNS > Off`.

As a last resort, you may need to try connecting to a different network, or contacting your ISP/Mobile carrier to find out if Meower is blocked.

**Comment by MikeDev101** [comment]
This appears to be a DNS-related issue on Firefox. I have also encountered this issue before, but primarily on desktop Firefox. Purging the DNS cache appears to help, also using CORS everywhere appears to help. I'm not sure how to repro on mobile.

**Comment by tnix100** [comment]
> This appears to be a DNS-related issue on Firefox. I have also encountered this issue before, but primarily on desktop Firefox. Purging the DNS cache appears to help, also using CORS everywhere appears to help. I'm not sure how to repro on mobile.

Hmm, never experienced this problem before. But I also wouldn't suggest using CORS everywhere since this can allow some CORS vulnerabilities and if you don't fully know what CORS is I'd not touch it at all. CORS everywhere also doesn't work on mobile.

One thing that'd help diagnose this problem further would be to copy the "TypeError: NetworkError" part and send it here. That'll tell us if it was an error that happened on the server or something going wrong before your request reaches the server.

**Comment by doesnmisclown** [comment]
> TypeError: NetworkError when attempting to fetch resource.

**Comment by JoshAtticus** [comment]
> > TypeError: NetworkError when attempting to fetch resource.

Hello @doesnmisclown, do you mind giving us an update? Have you tried following my steps and MikeDEV's steps above? If I don't get a response in 7 days, I'll close this issue.

**Comment by doesnmisclown** [comment]
Purging cache seems not work
I'm not use any proxy or VPN
and i check demo now: error still happens

**Comment by showierdata9978** [comment]
can you send  a log of the requests & console?

please send this to contact@meower.org as this has your plaintext password. we can filter it out

**Comment by tnix100** [comment]
> can you send a log of the requests & console?
> 
> please send this to [contact@meower.org](mailto:contact@meower.org) as this has your plaintext password. we can filter it out

I am going to say not to send your logs in plaintext, try to filter out any sensitive information you can find before sending. Email is not the place to send sensitive info and none of the Meower team should know your password.

**Comment by showierdata9978** [comment]
yeah true, but im warning them to not even send it on gh as that can be seen by anyone

**Comment by doesnmisclown** [comment]
how to open console on mobile?

**Comment by tnix100** [comment]
> how to open console on mobile?

Being on mobile makes it a lot harder to debug. Are you able to access https://api.meower.org in a new tab?

**Comment by showierdata9978** [comment]
https://github.com/liriliri/eruda

**Comment by doesnmisclown** [comment]
Oh chrome i can inject eruda using `javascript:`
On firefox it goes to search

**Comment by showierdata9978** [comment]
you need to type the javascript: part on ff manually

**Comment by doesnmisclown** [comment]
https://github.com/meower-media-co/Meower-Svelte/assets/116358854/39dd143b-2ecc-4587-9060-898dcc4b012b

**Comment by showierdata9978** [comment]
it looks like you edited the url

**Comment by doesnmisclown** [comment]
> it looks like you edited the url

wdym?

**Comment by JoshAtticus** [comment]
You can also follow [this guide](https://firefox-source-docs.mozilla.org/devtools-user/about_colon_debugging/index.html) if you have a Windows, Mac or Linux computer

It requires ADB debugging and a USB cable, if you don't know how to enable ADB debugging, follow [this guide](https://www.embarcadero.com/starthere/xe5/mobdevsetup/android/en/enabling_usb_debugging_on_an_android_device.html)
